misc/e2e_tests/docker/indexer/docker-compose.yml

```diff
@@ -3,8 +3,8 @@ version: '3'
 services:    
   e2e:    
     build:    
-      context: ../.
-      dockerfile: ./misc/Dockerfile    
+      context: ../../../../.
```

Comment:
This is now getting very very deeply nested and given the importance of E2E tests, suggests a new placement in my mind's eye. Maybe christen a new top-level directory `e2e_tests` or nestle `docker` right under `./test` ?